backend/database_wrapper.py

```diff
@@ -298,8 +298,9 @@ def get_subfolders_to_entries() -> List[Folder]:
 
 # --- Clip ---
 
-def create_clip(fid: int, name: str, video_format: str, start_time: timezone.datetime, end_time: timezone.datetime,
-                latitude: Decimal, longitude: Decimal, width: int, height: int, frame_rate: float) -> int:
+def create_clip(fid: int, name: str, video_format: str, start_time: timezone.datetime,
```

Comment:
Rename parameter name to clip_name to avoid confusing it with camera_name.
Note: only parameter not the models name.